### PR TITLE
Add "number of accumulations"

### LINKED ISSUE #30: [NTR] add "number of accumulations"
## Preferred term label

"number of accumulations"

## Synonyms

"number of acquisitions"

## Textual definition

The number of acquisitions on the same location, without position change. (Relevant to improve the S/N ratio.)

## Suggested parent term

Please look in the hierarchy in a browser such as the [NFDI4Chem TS](https://terminology.nfdi4chem.de/ts/ontologies/vibso)

## Attribution

ORCID:0000-0001-7694-5519

### PR DESCRIPTION
**Pull request by Zack-83**
Closes #30

### CODE REVIEW

**Review by StroemPhi — CHANGES_REQUESTED** [review]
@Zack-83, as you seem to have mentioned in https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/30#issuecomment-1422287625, I would rather just change the label and adjust the textual definition of VIBSO_0000007 instead of minting a new term, as this was what what actually meant from my side behind this term.

**Comment by Zack-83** [comment]
Fine for me. The numbers 00000xx are in fact arbitrary, until we publish :)

File: src/templates/vibso_classes.tsv [hunk 1]
```diff
@@ -7,7 +7,7 @@ obo:VIBSO_0000003	preamplifier gain setting		https://github.com/NFDI4Chem/Vibrat
...
 obo:VIBSO_0000006	device temperature setting		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/43	A setting datum that specifies the operating temperature of a device.		owl:Class	setting datum	ORCID:0000-0002-1595-3213		This class should better be defined in a more general device ontology.			
-obo:VIBSO_0000007	number of scans setting		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/44	A setting datum that specifies the number of iterations performed in a spectroscopic meassurement from which an aggregated assay output is generated.		owl:Class	setting datum|('part of' some 'vibrational spectrometer configuration specification')	ORCID:0000-0002-1595-3213	The setting entered into a Raman microscope which specifies the desired number of measurements this Raman microscope has to perform to produce an usable Raman spectrum.	This class should better be defined in a more general device ontology.			
+obo:VIBSO_0000007	number of accumulations	number of acquisitions https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/30
```
Comment (Zack-83) [inline src/templates/vibso_classes.tsv:10]:
```suggestion
obo:VIBSO_0000007	number of accumulations	number of acquisitions https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/30	A setting datum that specifies the number of iterations performed in a spectroscopic measurement from which an aggregated assay output is generated. owl:Class setting datum|('part of' some 'vibrational spectrometer configuration specification') ORCID:0000-0002-1595-3213 The setting entered into a Raman microscope which specifies the desired number of measurements this Raman microscope has to perform to produce an usable Raman spectrum.
```

**Review by StroemPhi — APPROVED** [review]

**Comment by StroemPhi** [comment]
@Zack-83 & @RainerStosch if you two are now fine with the definition:
`A setting datum that specifies the desired number of measurements a spectrometer has to perform to produce an usable spectrum.` then we can merge.

**Comment by StroemPhi** [comment]
But please let me do the merge.

**Comment by Zack-83** [comment]
Fine for me.

**Comment by Zack-83** [comment]
Your definition sounds more general. Ours is possibly more precise. Feel free to find a suitable compromise :)

**Pull request merged**